### PR TITLE
Updates page titles

### PR DESCRIPTION
**Pull request by DanielCouzens**
Please see ticket [116](https://github.com/epimorphics/standard-reports-ui/issues/116)

### CODE REVIEW

File: app/views/download_report/bad_request.html.haml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+= content_for(:title, "Invalid request".html_safe)
```
Comment (jonrandahl) [inline app/views/download_report/bad_request.html.haml]: This doesn't need `.html_safe` as there's nothing to make safe

File: app/views/download_report/bad_request.html.haml [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
 %h1.heading-large Invalid request
```
Comment (jonrandahl) [inline app/views/download_report/bad_request.html.haml]: this should be like the other pages, where we use `= yield(:title)` on the next line, to dry the code out

File: app/views/report_design/review_report.html.haml [hunk 3]
```diff
@@ -1,3 +1,5 @@
+= content_for(:title, "Your report file".html_safe)
```
Comment (jonrandahl) [inline app/views/report_design/review_report.html.haml]: This also doesn't need `.html_safe`

**Review by jonrandahl — CHANGES_REQUESTED** [review]
Small minor adjustments here, let me know if you have any questions!

File: app/views/download_report/bad_request.html.haml (hunk 2 above)
Comment (jonrandahl) [reply app/views/download_report/bad_request.html.haml]: This also doesn't need `.html_safe` either

File: app/views/download_report/bad_request.html.haml (hunk 1 above)
Comment (DanielCouzens) [reply app/views/download_report/bad_request.html.haml]: :+1:

File: app/views/download_report/bad_request.html.haml (hunk 2 above)
Comment (DanielCouzens) [reply app/views/download_report/bad_request.html.haml]: :+1:

File: app/views/report_design/review_report.html.haml (hunk 3 above)
Comment (DanielCouzens) [reply app/views/report_design/review_report.html.haml]: :+1:

**Review by jonrandahl — APPROVED** [review]
LGTM! 🚀

**Pull request merged**